compactor/src/compact.rs

```diff
@@ -324,67 +340,104 @@ impl Compactor {
         Ok(())
     }
 
-    /// Returns a list of partitions that have level 0 files to compact along with some summary
-    /// statistics that the scheduler can use to decide which partitions to prioritize. Orders
-    /// them by the number of level 0 files and then size.
-    pub async fn partitions_to_compact(&self) -> Result<Vec<PartitionCompactionCandidate>> {
-        let mut candidates = vec![];
+    /// Return a list of the most recent highest ingested throughput partitions.
+    /// The highest throughput partitions are prioritized as follows:
+    ///  1. If there are partitions with new writes within the last 4 hours, pick them.
```

Comment:
Also through here, I think this "write" means "write of a parquet file to object storage" rather than `DmlWrite`:

```suggestion
    ///  1. If there are partitions with new persisted files within the last 4 hours, pick them.
```

Reply:
Will replace `writes` with `ingested files`